File: app/src/main/java/com/example/projectone/AboutActivity.java
package com.example.projectone;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;
import android.widget.ImageView;

import androidx.appcompat.app.AppCompatActivity;

public class AboutActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_about);

        ImageView backButton = findViewById(R.id.back);

        backButton.setOnClickListener(v -> {
            // Go back to the previous activity
            onBackPressed();
        });

    }
}

File: app/src/main/java/com/angelodev/helpapp/ui/about/AboutActivity.kt
package com.angelodev.helpapp.ui.about

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.databinding.DataBindingUtil
import com.angelodev.helpapp.R
import com.angelodev.helpapp.databinding.ActivityAboutBinding
import dagger.hilt.android.AndroidEntryPoint

@AndroidEntryPoint
class AboutActivity : AppCompatActivity() {

    private lateinit var binding: ActivityAboutBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = DataBindingUtil.setContentView(this, R.layout.activity_about)

        binding.back.setOnClickListener {
            onBackPressedDispatcher.onBackPressed()
        }
    }
}
